scipy/signal/_peak_finding.py

```diff
@@ -1225,10 +1225,11 @@ def find_peaks_cwt(vector, widths, wavelet=None, max_distances=None,
         Minimum length a ridge line needs to be acceptable.
         Default is ``cwt.shape[0] / 4``, ie 1/4-th the number of widths.
     min_snr : float, optional
-        Minimum SNR ratio. Default 1. The signal is the value of
-        the cwt matrix at the shortest length scale (``cwt[0, loc]``), the
-        noise is the `noise_perc`th percentile of datapoints contained within a
-        window of `window_size` around ``cwt[0, loc]``.
+        Minimum SNR ratio. Default 1. The signal is the maximum CWT co-efficient
+        on the largest ridge line
+        (``cwt[ridge_line[0, 0], ridge_line[1, 0]]``).
+        The noise is `noise_perc`th percentile of datapoints contained within
+        the same ridge line (``noises[ridge_line[1, 0]]``).
```

Comment:
So I saw this in the paper and thought it would be a more precise way of describing the noise, since it describes the noise precisely as a value using the calculated values of the ridge line. The previous documentation used the `cwt` variable, which is not defined anywhere either, so I thought this would be more precise. You're right about the pseudo-code though, and I'll make changes to it.